lua/telescope/builtin/init.lua

```diff
@@ -74,6 +74,7 @@ builtin.grep_string = require_on_exported_call("telescope.builtin.__files").grep
 ---@param opts table: options to pass to the picker
 ---@field cwd string: root dir to search from (default: cwd, use utils.buffer_dir() to search relative to open buffer)
 ---@field find_command function|table: cmd to use for the search. Can be a fn(opts) -> tbl (default: autodetect)
+---@field file_entry_encoding string: encoding of output of `find_command`
```

Comment:
Why is the type a string, option works like a boolean. description could also be: "whether to re encode output of `find_command` to utf8" or something similar

Reply:
It is encoding name for ```vim.iconv```.  See ```help iconv```:
```
iconv({string}, {from}, {to})
    The result is a String, which is the text {string} converted
    from encoding {from} to encoding {to}.
```

Reply:
oh i misslocked, i thought it was used like a bool in the if. my bad